yaptide/batch/batch_methods.py

```diff
@@ -224,22 +225,31 @@ def prepare_script_files(payload_dict: dict, job_dir: str, sim_id: int, update_k
 
     backend_url = os.environ.get("BACKEND_EXTERNAL_URL", "")
 
-    submit_script = SUBMIT_SHIELDHIT.format(array_options=array_options,
-                                            collect_options=collect_options,
-                                            root_dir=job_dir,
-                                            n_tasks=str(payload_dict["ntasks"]),
-                                            convertmc_version=pymchelper.__version__)
-    array_script = ARRAY_SHIELDHIT_BASH.format(array_header=array_header,
-                                               root_dir=job_dir,
-                                               sim_id=sim_id,
-                                               update_key=update_key,
-                                               backend_url=backend_url)
-    collect_script = COLLECT_BASH.format(collect_header=collect_header,
+    if payload_dict['sim_type'] == SimulationType.FLUKA.value:
+        submit_template = SUBMIT_FLUKA
+        array_template = ARRAY_FLUKA_BASH
+        collect_template = COLLECT_FLUKA_BASH
+    elif payload_dict['sim_type'] == SimulationType.SHIELDHIT.value:
```

Comment:
maybe would be nice to add `else` clause or some other method to attach some default values to templates. You can for example set `submit_template = ""`
That would help future development when third option is added a simulation type